packages/nimble-components/tsconfig.json

```diff
@@ -13,7 +13,13 @@
         "resolveJsonModule": true,
         "sourceMap": true,
         "inlineSources": true,
-        "lib": ["DOM", "ES2015", "ES2016.Array.Include", "ES2017.Object"],
+        "lib": [
+            "DOM",
+            "DOM.Iterable",
```

Comment:
Added `DOM.Iterable` since I wanted the `DOMTokenList` type to support iteration. I don't think there's any reason not to.

Reply:
The spright and storybook tsconfigs should be in sync